experiment/references.md

```diff
@@ -1 +1,3 @@
-### Link your references in here
\ No newline at end of file
+- [https://en.wikipedia.org/wiki/Tuned_mass_damper](http://en.wikipedia.org/wiki/Tuned_mass_damper)
+- [https://en.wikipedia.org/wiki/Base_isolation.](http://en.wikipedia.org/wiki/Base_isolation.)
+- [https://en.wikipedia.org/wiki/Vibration_control](http://en.wikipedia.org/wiki/Vibration_control)
```

Comment:
1. Please use https links only.
2. Please use a descriptive phrase for the description instead of using the link.